stdlib/public/SDK/Foundation/Decimal.swift

```diff
@@ -205,61 +205,69 @@ extension Decimal : ExpressibleByIntegerLiteral {
 
 extension Decimal : SignedNumeric {
   public var magnitude: Decimal {
-    return Decimal(
-      _exponent: self._exponent, _length: self._length,
-       _isNegative: 0, _isCompact: self._isCompact,
-       _reserved: 0, _mantissa: self._mantissa)
+      return Decimal(
+          _exponent: self._exponent, _length: self._length,
+          _isNegative: 0, _isCompact: self._isCompact,
+          _reserved: 0, _mantissa: self._mantissa)
   }
 
   // FIXME(integers): implement properly
   public init?<T : BinaryInteger>(exactly source: T) {
-    fatalError()
+      fatalError()
   }
 
   public static func +=(_ lhs: inout Decimal, _ rhs: Decimal) {
       var rhs = rhs
-      NSDecimalAdd(&lhs, &lhs, &rhs, .plain)
+      withUnsafeMutablePointer(to: &lhs) {
```

Comment:
Thanks for taking care of these, they have been a thorn for a while now and possibly were causing some bugs.